Lambdas/GHHooks/EventHandler/Handlers/DocsIssuer.swift

```diff
@@ -0,0 +1,135 @@
+                    "Will not file issue for docs push PR because no docs files are added or modified",
+                    metadata: ["number": .stringConvertible(pr.number)]
+                )
+                continue
+            }
+            try await fileIssue(number: pr.number)
+        }
+    }
+
+    /// Should not contain any labels that indicate no need for a new issue.
+    func isNotExemptFromNewIssue(pr: SimplePullRequest) -> Bool {
+        Set(pr.knownLabels).intersection([.translationUpdate, .noTranslationNeeded]).isEmpty
+    }
+
+    func getPRsRelatedToCommit() async throws -> [SimplePullRequest] {
+        let response = try await context.githubClient.repos_list_pull_requests_associated_with_commit(
+            .init(path: .init(
+                owner: repo.owner.login,
+                repo: repo.name,
+                commit_sha: commitSHA
+            ))
+        )
+
+        guard case let .ok(ok) = response,
+              case let .json(json) = ok.body else {
+            throw Errors.httpRequestFailed(response: response)
+        }
+
+        return json
+    }
+
+    func getPRFiles(number: Int) async throws -> [DiffEntry] {
+        let response = try await context.githubClient.pulls_list_files(.init(
+            path: .init(
+                owner: repo.owner.login,
+                repo: repo.name,
+                pull_number: number
+            )
+        ))
+
+        guard case let .ok(ok) = response,
+              case let .json(json) = ok.body else {
+            throw Errors.httpRequestFailed(response: response)
+        }
+
+        return json
+    }
+
+    func fileIssue(number: Int) async throws {
+        let response = try await context.githubClient.issues_create(.init(
+            path: .init(
+                owner: repo.owner.login,
+                repo: repo.name
+            ),
+            body: .json(.init(
+                title: .case1(self.makeIssueTitle(number: number)),
+                body: self.makeIssueDescription(number: number)
+            ))
+        ))
+
+        guard case .created = response else {
+            throw Errors.httpRequestFailed(response: response)
+        }
+    }
+
+    func makeIssueTitle(number: Int) -> String {
+        "Translation needed for #\(number)"
+    }
+
+    func makeIssueDescription(number: Int) -> String {
+        return """
+        ---
+        title: Translation needed for #\(number)
+        ---
+
+        The docs have been updated in PR #\(number). The translations should be updated if required.
+
+        Languages:
+        - [ ] English
+        - [ ] Chinese
+        - [ ] German
+        - [ ] Dutch
+        - [ ] Italian
+        - [ ] Spanish
+        - [ ] Polish
+        - [ ] Korean
+
+        Assigned to @vapor/translators - please submit a PR with the relevant updates and check the box once merged. Please ensure you tag your PR with the `translation-update` so it doesn't create a new issue!
+        """
+    }
```

Comment:
Yeah, no, I gotta put my foot down on this one. Even back when I was just tinkering with the existing workflow Tim pointed out that this needs to be a template (and in the docs repo) for the sake of others who shouldn't have to be able to parse Penny's code just to update the translation list (it happens more than you'd think).